db/seed_helper.rb

```diff
@@ -1,13 +1,25 @@
 module SeedHelper
+  # unique_code has unique, not null contraint but cannot be given
+  # expected value until all offences seeded
+  # - see DataMigrator::OffenceUniqueCodeMigrator
+  def self.find_or_create_scheme_10_offence!(attrs)
```

Comment:
**[nit pick/not a blocker]** If all the methods here are `def self.method_name` wouldn't make more sense for this to be a **class** instead of a **module**?

FYI, alternatively you could consider keeping this as a **module** and allowing the module to be `extended/included`, as the _self_ definitions prevents that (although arguably this _module_ doesn't seem to be required anywhere else, hence preferring my first suggestion):

```ruby
module SeedHelper
  def find_or_create_scheme_10_offence
    ...
  end
  module_function :find_or_create_scheme_10_offence
end
```

_module_ seems to be overused in the code 😓 even when it doesn't bring any abstraction/behaviour sharing.

Reply:
its required in quite a few of the existing seed files if you search for `require Rails.root.join('db','seed_helper')`

Reply:
will give this ago, but feels like yak-shaving of existing functionality